tests/scripts/thread-cert/border_router/MATN_16_LargeNumberOfMulticastGroupSubscriptionsToBBR.py

```diff
@@ -161,7 +161,7 @@ def verify(self, pv: pktverify.packet_verifier.PacketVerifier):
             # 2.04 changed
             # Where the payload contains:
             # Status TLV: ST_MLR_SUCCESS
-            pkts.filter_wpan_src64(vars['BR_1']) \
+            pkts.copy().filter_wpan_src64(vars['BR_1']) \
```

Comment:
If this is necessary, should do the same for other scripts:
- MATN_02_MLRFirstUse.py
- test_bmlr.py

Reply:
I updated MATN_02 but I didn't find a similar case in test_bmlr.py.

Reply:
Should be fine since we already increased `AUTO_SEEK_BACK_MAX_DURATION`.